### ISSUE TITLE
WCAG 4.1.2 test 7: false positive

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#132]
[This document](https://github.com/veraPDF/veraPDF-wcag-algs/files/7736372/test-document-8.pdf) fails **WCAG 4.1.2 test 7** ("_Invalid Table header_"). It's particularly the occurrences on **page 52** that appear to be false positives, as these are texts that clearly have nothing to do with tables.

**Comment by bdoubrov** [comment]
This error is caused by incorrect parsing of marked content sequences within Form XObjects. To be fixed.
